charts/handbrake/2.0.0/questions.yaml

```diff
@@ -0,0 +1,797 @@
+                        default: 5900
+                        editable: false
+                        hidden: true
+                    - variable: nodePort
+                      label: "(optional) host nodePort to expose to"
+                      description: "only get used when nodePort is selected"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 36052
+                        required: true
+
+
+# Privileged
+  - variable: securityContext
+    group: "Storage and Devices"
+    label: "Security Context"
+    schema:
+      type: dict
+      attrs:
+        - variable: privileged
+          label: "Enable privileged to pass thru an Optical Drive device"
+          schema:
+            type: boolean
+            default: true
+            required: false
+
+# TrueCharts Specific
+  - variable: appVolumeMounts
+    label: ""
+    group: "Storage and Devices"
+    schema:
+      type: dict
+      attrs:
+          # config
+        - variable: config
+          label: "config Volume"
+          schema:
+            type: dict
+            $ref:
+              - "normalize/ixVolume"
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: setPermissions
+                label: "Automatic Permissions"
+                description: "Automatically set permissions on install"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: false
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: datasetName
+                label: "Name"
+                schema:
+                  type: string
+                  default: "config"
+                  editable: false
+              - variable: mountPath
+                label: "Mount Path inside the container"
+                schema:
+                  type: path
+                  default: "/config"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "config - Enable Host Path"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "config Host Path"
+                      description: "Host path to store config"
+                      schema:
+                        type: hostpath
+                        required: true
```

Comment:
```suggestion
# TrueCharts Specific
  - variable: appVolumeMounts
    label: "app storage"
    group: "Storage and Devices"
    schema:
      type: dict
      attrs:
        # Config ------------------------
        - variable: config
          label: "config dataset"
          schema:
            type: dict
            $ref:
              - "normalize/ixVolume"
            attrs:
              - variable: enabled
                label: "Enabled"
                schema:
                  type: boolean
                  default: true
                  required: true
                  hidden: true
                  editable: false
              - variable: setPermissions
                label: "Automatic Permissions"
                description: "Automatically set permissions on install"
                schema:
                  type: boolean
                  default: true
                  hidden: false
              - variable: emptyDir
                label: "emptyDir"
                schema:
                  type: boolean
                  default: false
                  hidden: true
                  editable: false
              - variable: datasetName
                label: "Dataset Name"
                schema:
                  type: string
                  default: "config"
                  required: true
                  editable: false
              - variable: mountPath
                label: "Mount Path"
                description: "Path to mount inside the pod"
                schema:
                  type: path
                  required: true
                  default: "/config"
                  editable: false
              - variable: hostPathEnabled
                label: "host Path Enabled"
                schema:
                  type: boolean
                  default: false
                  show_subquestions_if: true
                  subquestions:
                    - variable: hostPath
                      label: "Host Path"
                      schema:
                        type: hostpath
                        required: true
```

Standardize storage part1
(splitting because github dont like big suggestions)